README.md

```diff
@@ -25,6 +25,11 @@ This repository holds the kcp tests that tests against the publicly available  i
   ```console
   $ kubectl oidc-login get-token --oidc-issuer-url=<oidc issuer url> --oidc-client-id=<oidc client ID> --oidc-redirect-url-hostname=127.0.0.1
   ```
+  - For kcp services deploy with redhat sso( **Only supported kcp services deploy with redhat sso** ), there's another login method by use `offline token`, you could get an offline token ( [How to generate an offline token](https://access.redhat.com/articles/3626371#bgenerating-a-new-offline-tokenb-3) ) and log in to the kcp service by execute the follow commands.
+    ```console
+    $ export OFFLINE_TOKEN=<YOUR_OFFLINE_TOKEN>
+    $ make login
+    ```
```

Comment:
@Phaow or should we say as below ?
please make sure to export OFFLINE_TOKEN and run make login before running the smoke tests if kcp services have been deployed with redhat SSO ?

Reply:
Hmm, from my understanding we support both 2 ways use offline token or previous way username/password, so it not a must step for running kcp smoke test just a convenient way to not open a browser input username/password. So I think we seems no need to add the point. WDYT?

Reply:
@Phaow make sense and agree, Just added few minor comments and once addressed we are good to be merged, thanks !!